.github/workflows/Pyodide.yml

```diff
@@ -0,0 +1,72 @@
+name: DuckDB-Pyodide
+on:
+  workflow_call:
+    inputs:
+      override_git_describe:
+        type: string
+      git_ref:
+        type: string
+      skip_tests:
+        type: string
+  workflow_dispatch:
+    inputs:
+      override_git_describe:
+        type: string
+      git_ref:
+        type: string
+      skip_tests:
+        type: string
+  repository_dispatch:
+  push:
+    branches:
+      - "**"
+      - "!main"
+      - "!feature"
+    paths-ignore:
+      - "**.md"
+      - "examples/**"
+      - "test/**"
+      - "tools/**"
+      - "!tools/pythonpkg/**"
+      - ".github/patches/duckdb-wasm/**"
+      - ".github/workflows/**"
+      - "!.github/workflows/Pyodide.yml"
+
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+    paths-ignore:
+      - "**.md"
+      - "examples/**"
+      - "test/**"
+      - "tools/**"
+      - "!tools/pythonpkg/**"
+      - ".github/patches/duckdb-wasm/**"
+      - ".github/workflows/**"
+      - "!.github/workflows/Pyodide.yml"
+
+jobs:
+  build_pyodide:
+    name: Build pyodide wheel
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - python: "3.10"
+            pyodide-build: "0.22.1"
+            emscripten-sdk: "3.1.27"
+          - python: "3.11"
+            pyodide-build: "0.25.1"
+            emscripten-sdk: "3.1.46"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.version.python }}
+      - run: pip install 'pyodide-build==${{ matrix.version.pyodide-build }}' 'pydantic<2'
+      - uses: mymindstorm/setup-emsdk@v14
+        with:
+          version: ${{ matrix.version.emscripten-sdk }}
+
+      - run: pyodide build --exports=whole_archive
+        working-directory: ./tools/pythonpkg
```

Comment:
Can the built artifact be saved, so they are available to be downloaded?

Something along the lines of:
```
      - uses: actions/upload-artifact@v3
        with:
          name: java-linux-amd64
          path: |
            build/release/tools/jdbc/duckdb_jdbc.jar
```
(this is from Java.yml)